game/templates/game/landing.html

```
@@ -48,7 +48,17 @@
     <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
     <link rel="stylesheet" href="{% static 'game/css/landing.css' %}">
+    <script>
+        (function () {
+            const VALID_PAGE_THEMES = ['light', 'dark'];
+            let savedPageTheme = 'dark';
+            try { savedPageTheme = localStorage.getItem('pageTheme') || 'dark'; } catch (e) { }
+            if (!VALID_PAGE_THEMES.includes(savedPageTheme)) savedPageTheme = 'dark';
+            document.documentElement.setAttribute('data-page-theme', savedPageTheme);
+        })();
+    </script>>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)